test/e2e/tests/security-provider.spec.js

```diff
@@ -1,235 +0,0 @@
-const { strict: assert } = require('assert');
-const { convertToHexValue, withFixtures, openDapp } = require('../helpers');
-const FixtureBuilder = require('../fixture-builder');
-
-const OPENSEA_URL =
-  'https://proxy.metafi.codefi.network/opensea/security/v1/validate';
-
-/**
- * @param {import('mockttp').Mockttp} mockServer - The mock server.
- */
-
-describe('Transaction security provider', function () {
```

Comment:
We are still supporting this in main.. I don't think that we should remove this tests? Was there any reason for removing them?